.readthedocs.yml

```diff
@@ -0,0 +1,19 @@
+conda:
```

Comment:
Does this line (`conda:`) do anything?

Reply:
https://docs.readthedocs.io/en/stable/config-file/v2.html#conda is why I'm thinking it might not